pytest_django/live_server_helper.py

```diff
@@ -24,8 +24,11 @@ def __init__(self, addr):
                 and conn.settings_dict["NAME"] == ":memory:"
             ):
                 # Explicitly enable thread-shareability for this connection
-                conn.allow_thread_sharing = True
-                connections_override[conn.alias] = conn
+                try:
+                    conn.allow_thread_sharing = True
+                    connections_override[conn.alias] = conn
+                except AttributeError:
+                    pass
```

Comment:
It should use `inc_thread_sharing()` with Django 2.2+ I think (https://github.com/django/django/commit/76990cbbda5d93fda560c8a5ab019860f7efaab7).